src/test/java/com/capitalone/dashboard/collector/PullRequestCollectorTest.java

```diff
@@ -54,14 +54,15 @@
 
     @Test
     public void testGetPullRequests() throws IOException, URISyntaxException, EncryptionException {
-        given(settings.getKey()).willReturn("abcdefghijklmnopqrstuvwxyz1234567");
-        String encPassword = Encryption.encryptString("password", settings.getKey());
-        given(settings.getPassword()).willReturn(encPassword);
+        //given(settings.getKey()).willReturn("abcdefghijklmnopqrstuvwxyz1234567");
```

Comment:
please remove the commented lines of code

Reply:
done